utils/setup.php

```diff
@@ -413,14 +441,23 @@
         fail(pg_last_error($aDBInstances[$iLoadThreads]->connection));
     }
 
-    $bAnyBusy = true;
-    while ($bAnyBusy) {
-        $bAnyBusy = false;
-        for ($i = 0; $i <= $iLoadThreads; $i++) {
-            if (pg_connection_busy($aDBInstances[$i]->connection)) $bAnyBusy = true;
+    $failed = false;
+    for ($i = 0; $i <= $iLoadThreads; $i++) {
+        while (($pgresult = pg_get_result($aDBInstances[$i]->connection)) !== false) {
+            $resultStatus = pg_result_status($pgresult);
+            // PGSQL_EMPTY_QUERY, PGSQL_COMMAND_OK, PGSQL_TUPLES_OK,
+            // PGSQL_COPY_OUT, PGSQL_COPY_IN, PGSQL_BAD_RESPONSE,
+            // PGSQL_NONFATAL_ERROR and PGSQL_FATAL_ERROR
+            echo 'Query result ' . $i . ' is: ' . $resultStatus . '\n';
+            if ($resultStatus != PGSQL_COMMAND_OK && $resultStatus != PGSQL_TUPLES_OK) {
+                $resultError = pg_result_error($pgresult);
+                echo '-- error text ' . $i . ': ' . $resultError . '\n';
+                $failed = true;
+            }
         }
-        sleep(1);
-        echo '.';
+    }
+    if ($failed) {
+        fail('SQL errors loading placex and/or location_property_osmline tables');
```

Comment:
During development of this change, my first attempt at running the `--load-data` steps failed due to errors that I later resolved. The errors were difficult to resolve because the original implementation of this block never checked the completion status of the `INSERT INTO placex …` or `insert into location_property_osmline …` statements, and never reported any errors resulting from those queries. I implemented proper status and error checking and reporting in order to discover the errors, and left that code in the PR because it was a clear improvement to the existing implementation.